docs/guides/mssqlserver/volume-expansion/volume-expansion.md

```diff
@@ -0,0 +1,264 @@
+
+
+Let's create the `MSSQLServerOpsRequest` CR we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubedb/docs/raw/{{< param "info.version" >}}/docs/example/mssqlserver/volume-expansion/mops-volume-exp-ag-cluster.yaml
+mssqlserveropsrequest.ops.kubedb.com/mops-volume-exp-ag-cluster created
+```
+
+#### Verify MSSQLServer volume expanded successfully
+
+If everything goes well, `KubeDB` Ops-manager operator will update the volume size of `MSSQLServer` object and related `PetSet` and `Persistent Volumes`.
+
+Let's wait for `MSSQLServerOpsRequest` to be `Successful`.  Run the following command to watch `MSSQLServerOpsRequest` CR,
+
+```bash
+$ kubectl get mssqlserveropsrequest -n demo
+NAME                         TYPE              STATUS       AGE
+mops-volume-exp-ag-cluster   VolumeExpansion   Successful   8m30s
+```
+
+We can see from the above output that the `MSSQLServerOpsRequest` has succeeded. 
+
+Now, we are going to verify from the `Petset`, and the `Persistent Volumes` whether the volume of the database has expanded to meet the desired state, Let's check,
+
+```bash
+$ kubectl get petset -n demo mssqlserver-ag-cluster -o json | jq '.spec.volumeClaimTemplates[].spec.resources.requests.storage'
+"2Gi"
+
+$ kubectl get pv -n demo
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+pvc-059f186a-01a4-441d-85f1-95aef34934be   2Gi        RWO            Delete           Bound    demo/data-mssqlserver-ag-cluster-0   longhorn       <unset>                          29m
+pvc-87bea35f-4a55-4aa5-903a-e4da9f548241   2Gi        RWO            Delete           Bound    demo/data-mssqlserver-ag-cluster-1   longhorn       <unset>                          29m
+pvc-9d1c3c9c-f928-4fa2-a2e1-becf2ab9c564   2Gi        RWO            Delete           Bound    demo/data-mssqlserver-ag-cluster-2   longhorn       <unset>                          29m
+```
+
+The above output verifies that we have successfully expanded the volume of the MSSQLServer database.
+
+## Standalone Mode
+
+The volume expansion process is same for all the MSSQLServer modes. The `MSSQLServerOpsRequest` CR has the same fields. The database needs to refer to a mssqlserver 
+in standalone mode.
+
+## Cleaning Up
+
+To clean up the Kubernetes resources created by this tutorial, run:
+
+
+```bash
+$ kubectl patch -n demo ms/mssqlserver-ag-cluster -p '{"spec":{"deletionPolicy":"WipeOut"}}' --type="merge"
+mssqlserver.kubedb.com/mssqlserver-ag-cluster patched
+
+$ kubectl delete -n demo mssqlserver mssqlserver-ag-cluster
+mssqlserver.kubedb.com "mssqlserver-ag-cluster" deleted
+
+$ kubectl delete -n demo mssqlserveropsrequest mops-volume-exp-ag-cluster
+mssqlserveropsrequest.ops.kubedb.com "mops-volume-exp-ag-cluster" deleted
```

Comment:
mssqlserveropsrequest.ops.kubedb.com "mops-volume-exp-ag-cluster" deleted ->
mssqlserveropsrequest.ops.kubedb.com/mops-volume-exp-ag-cluster deleted

Reply:
We should not change this,  As this is a confirmation message of `kubectl delete` command